google-map-point.html

```diff
@@ -0,0 +1,58 @@
+<!-- Copyright (c) 2015 Google Inc. All rights reserved. -->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../google-apis/google-maps-api.html">
+
+<!--
+The `google-map-point` element represents a point on a map. It's used as a child of other
+google-map-* elements.
+
+<b>Example</b>:
+
+    <google-map-point latitude="37.77493" longitude="-122.41942"></google-map-point>
```

Comment:
this doesnt show the point you made above, which is that this element needs to be a child of another map element :) Please include.